src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-sourceFolder.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <p>Optionally specify the location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. To generate a code coverage report, MATLAB uses only the source code in the specified folder and its subfolders. </p>
+    <p>Specify a source folder path in the input box. Click on 'Add folder' to add more values.</p>
```

Comment:
Paste the source folder path in the **File path** box. To add more folders, click **Add folder**.

Please confirm that **File path** and **Add folder** are correct.

Reply:
Done.